extension/src/completion/autocompletionProvider.ts

```diff
@@ -58,6 +60,19 @@ function readDataFileByLine(datafile: string, action: (items: string[]) => void)
         }
     });
 }
+
+// loads resources for the openWebHelp.ts objects. 
```

Comment:
we use this extension to format the TS codes:
https://marketplace.visualstudio.com/items?itemName=eternalphane.tsfmt-vscode